### PR TITLE
feat: update depedencies and lock the versions

### PR DESCRIPTION
**Pull request by Keith-CY**

### CODE REVIEW

**Comment by ashchan** [comment]
Restarted CI build.

**Comment by duanyytop** [comment]
Keys test case fail because of async

**Comment by Keith-CY** [comment]
CI passed

File: packages/neuron-wallet/src/services/transactions.ts [hunk 1]
```diff
@@ -290,7 +290,7 @@ export default class TransactionsService {
     const previousOutputs: OutputEntity[] = []
     for (const i of transaction.inputs!) {
       const input = new InputEntity()
-      input.outPointHash = i.previousOutput.hash
+      input.outPointTxHash = i.previousOutput.txHash
```
Comment (classicalliu) [inline packages/neuron-wallet/src/services/transactions.ts]: It's this version of SDK is still `outPoint.txHash` or `outPoint.cell.txHash` ?

Comment (Keith-CY) [reply packages/neuron-wallet/src/services/transactions.ts]: > It's this version of SDK is still `outPoint.txHash` or `outPoint.cell.txHash` ?

prerelease of v0.11, which use outpoint of `{ txHash, index }`

File: packages/neuron-wallet/package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged & yarn test"
```
Comment (classicalliu) [inline packages/neuron-wallet/package.json]: `&` or `&&`? But I think test can remove from `pre-commit`, CI will check it.

File: packages/neuron-wallet/src/services/transactions.ts (hunk 1 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/services/transactions.ts]: ok, i c

File: packages/neuron-wallet/package.json (hunk 2 above)
Comment (Keith-CY) [reply packages/neuron-wallet/package.json]: `&&` is correct.

CI usually blocked

**Pull request merged**